app/views/form/page.html.erb

```diff
@@ -40,12 +40,12 @@
       <%= f.hidden_field :page, value: @page.id %>
 
       <div class="govuk-button-group">
-        <% if !@page.id.include?("value_check") %>
+        <% if !@page.id.include?("value_check") && if request.query_parameters["referrer"] != "check_answers" %>
```

Comment:
Possibly getting complex enough to be worth extracting to a helper that has a spec?